ui/src/components/OrderModal.jsx

```diff
@@ -0,0 +1,32 @@
+import ParcelSizeSummaryCard from "./SummaryCards/ParcelSizeSummaryCard";
+import RecipientSummaryCard from "./SummaryCards/RecipientSummaryCard";
+import SenderSummaryCard from "./SummaryCards/SenderSummaryCard";
+import {Dialog} from "@material-ui/core"
+import "../styles/OrderModal.css"
+
+export default function OrderModal(props){
+    return(
+        <Dialog
+            open = {props.open}
+            onClose = {() => props.onClose(props.order)}
+        >
```

Comment:
Maybe set `border-radius: 0 ` to comply with the app's style

Reply:
Did global theme with border-radius 0